### PR TITLE
Adds all .rst files to build. 

### LINKED ISSUE #56: [BUG] - Missing CHANGELOG.rst from packaged tar.gz
**Describe the bug**
setup.py includes a dependency on CHANGELOG.rst which is not included in the .tar.gz packages

**To Reproduce**
Retrieve django-guid-<version>.tar.gz and try to run setup.py

I've tested this on 1.1.1, 2.0.0 2.1.0 & 2.20

**Full stack trace**
[root@dev-rhel7-acme-python-django2-guid django-guid-2.2.0]# /opt/acme/bin/python3 setup.py install
Traceback (most recent call last):
  File "setup.py", line 7, in <module>
    with open('CHANGELOG.rst') as changelog_file:
FileNotFoundError: [Errno 2] No such file or directory: 'CHANGELOG.rst'

### PR DESCRIPTION
**Pull request by JonasKs**
Closes #56 . 

This is a merge request from this bugfix branch into the `2.2`-branch. I will then release a `2.2.1` version from the `2.2`-branch, which will include `CHANGELOG.rst` in the `sdist`, solving #56. 

I'll create this as a draft and think about it for a while. Please comment if you have any comments or concerns about this approach.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+include *.rst
```
Comment (sondrelg) [inline MANIFEST.in:1]: Do you want all rst-files?

Comment (JonasKs) [reply MANIFEST.in:1]: It won't add `docs/*` only top level files ending in `.rst`. `sdist` already adds all tests, the README and so on automatically, so I didn't see any issue with adding the `CONTRIBUTORS` and `CONTRIBUTING` files as well.

**Comment by JonasKs** [comment]
I'll fix pipelines when we actually need to backport functionality and tests needs to run.

**Comment by JonasKs** [comment]
I'll fix pipelines when we actually need to backport functionality and tests needs to run.

**Pull request merged**